### PR TITLE
Add Docker tutorial

### PR DESCRIPTION
**Pull request by TheZoker**
This PR adds a tutorial on how to setup Docker, Docker Compose and a little Minecraft server on a Ubuntu server

I have read and understood the Contributor's Certificate of Origin at the end of the template and I hereby certify that I meet the contribution criteria described in it.

Signed-off-by: Florian Gareis dev@zoker.me

### CODE REVIEW

**Comment by TheZoker** [comment]
Just saw, that there is already a tutorial for this #12 🤦 

I'm keeping this PR open for now, maybe there is something useful in here. If the other tutorial is merged, I'll close this

File: community-tutorials/install-docker/01-en.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+title: Setup Docker on a Ubuntu server
```
Comment (raphi-0901) [inline community-tutorials/install-docker/01-en.md:2]: Please specify more precisely what you are addressing in this tutorial (docker-compose, minecraft-server) for title, slug, and description

**Review by raphi-0901 — CHANGES_REQUESTED** [review]

File: community-tutorials/install-docker/01-en.md (hunk 1 above)
Comment (TheZoker) [reply community-tutorials/install-docker/01-en.md]: Adjusted the title, description and slug to be more precise, thank you for the feedback 👍

**Comment by raphi-0901** [comment]
I have followed the tutorial and then tried to connect to the server. Unfortunately, this did not work. Do I need to configure anything else on the Docker image for this to work?

**Comment by TheZoker** [comment]
@raphi-0901 Did you use the Java or the UWP version of Minecraft to connect to the server? 
This tutorial only works with the Java version (I will add that to the tutorial requirements)

**Comment by TheZoker** [comment]
Ah I just realized, that I accidently used `/udp` instead of `/tcp`. Fixed 👍

**Pull request merged**